pyproject.toml

```diff
@@ -74,5 +75,5 @@ warn_unused_configs = true
 no_implicit_reexport = true
 
 [tool.black]
-line-length = 80
+line-length =  88
```

Comment:
Any reason for this change?

Reply:
In .flake8 and .editorconfig it is set to 88